monitoring/psi-prod-cluster/overlays/prod/prometheus.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  labels:
+    prometheus: aicoe-sre
+  name: aicoe-sre-prometheus
```

Comment:
I wonder if we should change the name.

Reply:
that would probably come in a different pr, this one is just a patch on the original base with the same name

Reply:
updated the name (might as well)